### ISSUE TITLE
Chrome 41: `Math.asinh` and `Math.atanh` are imprecise

### ISSUE THREAD

**Issue by ljharb** [paulmillr/es6-shim#334]
``` js
var isAsinhOK = Math.asinh(0.0002) >= 0.0001999999986666666 &&  Math.asinh(0.0002) <= 0.0001999999986666668;
var isAtanhOK = Math.atanh(0.0002) >= 0.0002000000026666667 && Math.atanh(0.0002) <= 0.0002000000026666668;
```

Both of these are true in shimmed Firefox 37 and Safari 8, both false in Chrome 41 and Chrome Canary 44.

From https://github.com/kangax/compat-table/issues/392#issuecomment-92308965

**Comment by ljharb** [comment]
@Yaffle Note that our actual `asinh` shim fails this test - the native ones don't.

**Comment by Yaffle** [comment]
This may help:

```

var sqrt = Math.sqrt;
var log1p = Math.log1p;
var expm1 = Math.expm1;

var LOG2E = Math.LOG2E
var E = Math.E;
var sign = Math.sign;
var abs = Math.abs;

Math.acosh = function (x) {
  x = Number(x);
  if (x < 1) {
    return 0 / 0;
  }
  if (x < 2) {
    return log1p(x - 1 + sqrt(x - 1) * sqrt(x + 1));
  }
  return log1p(x / 2 - 1 + sqrt(x - 1) * sqrt(x + 1) / 2) + 1 / LOG2E;
};
Math.asinh = function (x) {
  x = Number(x);
  var s = sign(x);
  var a = abs(x);
  if (a < 1) {
    return s * log1p(a + expm1(log1p(a * a) / 2));
  }
  return s * (log1p(a / 2 - 1 + sqrt(a + 1 / a) * sqrt(a) / 2) + 1 / LOG2E);
};
Math.atanh = function (x) {
  x = Number(x);
  var s = sign(x);
  var a = abs(x);
  return s * log1p(2 * a / (1 - a)) / 2;
};
Math.cosh = function (x) {
  x = Number(x);
  var a = abs(x);
  var t = expm1(a - 1) + 1;
  return (t + 1 / (t * E * E)) * (E / 2);
};
Math.sinh = function (x) {
  x = Number(x);
  var s = sign(x);
  var a = abs(x);
  if (a < 1) {
    var t = expm1(a);
    return s * t * (1 + 1 / (t + 1)) / 2;
  }
  var t = expm1(a - 1);
  return s * (t + 1 - 1 / ((t + 1) * E * E)) * (E / 2);
};
Math.tanh = function (x) {
  x = Number(x);
  var s = sign(x);
  var a = abs(x);
  if (a < 1) {
    var t = expm1(a);
    return s * t * (t + 2) / (t * t + 2 * t + 2);
  }
  return s / (1 + 2 / expm1(2 * a));
};
```

**Comment by anematode** [comment]
It looks like this issue is fixed; probably should close the issue?

**Comment by Yaffle** [comment]
yes

**Comment by ljharb** [comment]
When (or in what PR/commit) was it fixed?

**Comment by Yaffle** [comment]
https://github.com/paulmillr/es6-shim/commit/ec76d49cb801a03fb0a98041e2bba190a41b8288

**Comment by ljharb** [comment]
Thanks!

Fixed in ec76d49, then.
